class-post-by-email.php

```diff
@@ -726,7 +726,7 @@ protected function save_attachments( $uid, $postID ) {
 		foreach ( $map as $key => $value ) {
 			$p = $part->getPart( $key );
 
-			if ( 'attachment' == $p->getDisposition() ) {
+			if ( 'attachment' == $p->getDisposition() || ( 'inline' == $p->getDisposition() && 'image' == $p->getPrimaryType() ) ) {
```

Comment:
@codebykat This is the magic condition that tells us we're looking at an inlined image.